scripts/update-metadata.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# sync properties in layers config file with live meta data
+# Usage: ./scripts/update-metadata.sh <layerId (optional)>
+
+
+version=0.6.1
+workingDir="./download"
+layersConfigFile="../data/layers-config.json"
+layersConfig=$(cat $layersConfigFile)
+#keys=$(jq -r 'keys[]' <<< "$layersConfig" | head -n 1)
+
+# take optional layer ID from command line argument
+layerID=$1 
+
+# check if layer ID is set
+if [ -z $layerID ]
+then
+    keys=$(jq -r 'keys[]' <<< "$layersConfig")
```

Comment:
can't we just say `keys=$(jq -r 'keys[]' $layersConfigFile)` here?

Reply:
Yes, we could do this. But I am caching the content to save i/o operations probably overcautious?!

Reply:
Using the file directly for readability